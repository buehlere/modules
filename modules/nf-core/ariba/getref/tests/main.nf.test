// TODO nf-core: Once you have added the required tests, please run the following command to build this file:
// nf-core modules test ariba/getref
nextflow_process {

    name "Test Process ARIBA_GETREF"
    script "../main.nf"
    process "ARIBA_GETREF"

    tag "modules"
    tag "modules_nfcore"
    tag "ariba"
    tag "ariba/getref"

    // TODO nf-core: Change the test name preferably indicating the test-data and file-format used
    test("card reference") {

        // TODO nf-core: If you are created a test for a chained module
        // (the module requires running more than one process to generate the required output)
        // add the 'setup' method here.
        // You can find more information about how to use a 'setup' method in the docs (https://nf-co.re/docs/contributing/modules#steps-for-creating-nf-test-for-chained-modules).

        when {
            process {
                """
                // TODO nf-core: define inputs of the process here. Example:

                input[0] = "card"
                """
            }
        }

        then {
            assertAll(
                { assert process.success },
                { assert snapshot(file(process.out.db.get(0)).name).match("card.tar.gz") },
                //TODO nf-core: Add all required assertions to verify the test output.
            )
        }

    }

}
