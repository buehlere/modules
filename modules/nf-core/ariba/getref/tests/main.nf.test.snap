{
    "card.tar.gz": {
        "content": [
            "card.tar.gz"
        ],
        "timestamp": "2023-11-29T17:35:11.397206"
    }
}